repose-aggregator/tests/functional-test-framework/src/main/resources/default-test.properties

```diff
@@ -0,0 +1,47 @@
+#_=_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=
+#Repose
+#_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+#Copyright (C) 2010 - 2015 Rackspace US, Inc.
+#_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+#=_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
+
+# Properties files used by Groovy tests to support launching Repose and sending requests
+# to Repose
+
+test.root.directory=/default/test/root
+
+repose.version=${version}
+repose.home=/default/repose/home
+repose.config.directory=/configs
+repose.config.templates=/default/config/templates
+repose.log.name=/logs/repose.log
+repose.lint.log.name=/logs/repose-lint.log
+repose.log.pattern=/logs/repose_%d{yyyy-MM-dd_HHmmss}.log
+repose.container=/default/repose/container
+
+repose.jar=/default/valve/location
+repose.lint.jar=/default/lint/location
```

Comment:
anything where you are putting in a placeholder, just don't add. if you want to make an example of the other properties do that.